features/enlinkd/config/src/main/java/org/opennms/netmgt/config/EnhancedLinkdConfig.java

```diff
@@ -147,7 +147,7 @@ public interface EnhancedLinkdConfig {
     /**
      * <p>getConfiguration</p>
      *
-     * @return a {@link org.opennms.netmgt.config.linkd.LinkdConfiguration} object.
+     * @return a {@link org.opennms.netmgt.config.enlinkd.EnlinkdConfiguration} object.
```

Comment:
I wonder if we shouldn't remove comments which say the same as the code itself. In general I would only add comments if they add additional information.